### DISCUSSION TITLE
Sync notes in background on Android client when there is internet connectivity

### DISCUSSION THREAD

**Discussion by frego** [standardnotes/forum#2510] [Suggestions]
I have to always remember to launch Standard Notes before going offline to sync notes written on other devices. I, of course, often fail at it. It would be nice to be able a way to tell Standard Notes to sync when running in background.

**Comment by atmoio** [comment]
Hmm...will have to look into the feasibility of this on both platforms, iOS and Android. Would indeed be a nice addition.

**Comment by atmoio** [comment]
[Removed spam post]
